everestpy/src/everest/module.cpp

```diff
@@ -23,6 +23,8 @@ Module::Module(const RuntimeSession& session) : Module(get_variable_from_env("EV
 Module::Module(const std::string& module_id_, const RuntimeSession& session_) :
     module_id(module_id_), session(session_) {
 
+    Everest::Logging::init(session.get_logging_config_file().string(), module_id);
+
```

Comment:
This is debatable, since logging should not happen before this is called. There's one edge-case where logging might happen earlier: in the deprecated RuntimeSession ctor ManagerSettings could log if something goes wrong